ksqldb-engine/src/main/java/io/confluent/ksql/schema/ksql/inference/DefaultSchemaInjector.java

```diff
@@ -266,7 +267,24 @@ private static CreateSource addSchemaFields(
     final TableElements elements = buildElements(preparedStatement, keySchema, valueSchema);
 
     final CreateSource statement = preparedStatement.getStatement();
-    return statement.copyWith(elements, statement.getProperties());
+    final CreateSourceProperties properties = statement.getProperties();
+
+    Optional<String> avroKeySchemaName = Optional.empty();
```

Comment:
Nit: should we make this variable `final` and add an `else` block below to assign `Optional.empty()` ? (same for `avroValueSchemaName`)

Reply:
Sure.